### PR TITLE
Update the README file

### PR DESCRIPTION
**Pull request by ccamacho**
Hey Jiri, after a session using Inlunch I have ended doing some updates to your README file.
- Add some information to the README file. -

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,3 +70,23 @@ to deploy it on your workstation. (You'll still run
...
+  
+    ```bash
+  git clone https://github.com/openstack/tripleo-heat-templates
+  ```
+  Then deploy the overcloud:
+  
+  ```bash
+  openstack overcloud deploy \
+  --libvirt-type qemu \
+  --ntp-server clock.redhat.com \
```
Comment (jistr) [inline README.md]: This is an internal NTP server only. We could put in pool.ntp.org instead.

Comment (jistr) [inline README.md]: Also i wonder whether the overcloud deployment instructions should be part of tripleo-docs and we could just link to tripleo.org docs from here.

**Comment by ccamacho** [comment]
- Have changed the ntp server to use pool.
- I think the instructions are actually in tripleo.org, but there is no "just run this to create the overcoud". So a brief command intro might be useful here, and for further info "please go to tripleo.org", as it is, the docs will show you how to fully deploy tripleo using inlunch.

**Comment by jistr** [comment]
Ok that makes sense, thanks. The code looks good but can we squash it into a single commit instead of introducing the wrong value first and then fixing it? Currently the pull request contains 3 commits.

**Comment by ccamacho** [comment]
Fixed! :)

**Comment by jistr** [comment]
Perfect, thanks!

**Pull request merged**